### PR TITLE
adds `fasd-dired-function` to customize

### PR DESCRIPTION
**Pull request by FrancescElies**
When using fasd I would like to have a deer buffer instead of a dired one.

How do you feel about adding a variable to defcustom?

### CODE REVIEW

**Comment by FrancescElies** [comment]
I got some feedback froma very similar pr https://github.com/rejeep/prodigy.el/pull/110 therefore I updated this one as well

File: fasd.el [hunk 1]
```diff
@@ -37,6 +37,14 @@
 When set to nil, all fasd results are returned for completion"
   :type 'boolean)
 
+(defcustom fasd-file-manager 'dired
+  "A default set of file managers to use with `fasd-find-file'"
+  :group 'prodigy
```
Comment (steckerhalter) [inline fasd.el]: Is this intentional with the group `prodigy`?

Comment (FrancescElies) [reply fasd.el]: copy paste mistake, sorry :/

**Comment by FrancescElies** [comment]
> Right, but it's not supposed to be part of fasd.el. Can you remove it?

I'm not sure if I understand, this is related to https://github.com/steckerhalter/emacs-fasd/pull/13#discussion_r165702114, right? Now `:group 'prodigy` shouldn't be there any more

**Comment by steckerhalter** [comment]
Right, thanks.

**Pull request merged**